lib/data/multipleBackendGateway.js

```diff
@@ -77,9 +147,36 @@ const multipleBackendGateway = {
                     }
                     cb();
                 });
+            } else if (client.clientType === 'aws_s3') {
+                client.headBucket({ Bucket: client.awsBucketName },
+                err => {
+                    if (err) {
+                        multBackendResp[location] = { error: err };
```

Comment:
Ah good call. For the error, unless there's something wrong with AWS itself, it will return a 403 AccessDenied error, which the user will be able to see. And I didn't do versioning :)